py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -287,6 +287,15 @@ def scatter_add_decomposition(
     return scatter_add_tensor
 
 
+@register_torch_trt_decomposition(aten._log_softmax, registry=TORCH_TRT_DECOMPOSITIONS)
+def log_softmax_decomposition(
+    x: torch.Tensor,
+    dim: int,
+    half_to_float: bool,
```

Comment:
What is this arg for? 
Doesnt look like its part of https://pytorch.org/docs/stable/generated/torch.nn.functional.log_softmax.html

Reply:
It's in the schema: https://github.com/pytorch/pytorch/blob/b2386bdca1af4cd3eaf0b5b99ca26ca727bd57bb/aten/src/ATen/native/native_functions.yaml#L3692. `softmax` also has that arg but doesn't get used in our converter.

Reply:
If softmax has it, presumably this arg should be passed to it in the decomp pass and let the softmax converter decide behavior? That way the decision about how to deal with this parameter is centralized?